crates/widgets/src/widget/header.rs

```diff
@@ -150,88 +150,100 @@ where
         renderer: &Renderer,
         clipboard: Option<&dyn Clipboard>,
     ) {
-        let child_len = self.children.len();
-        let start_offset = if self.left_margin { 1 } else { 0 };
-        let end_offset = if self.right_margin { 1 } else { 0 };
-
-        let dividers = self
-            .children
-            .iter()
-            .enumerate()
-            .zip(layout.children())
-            .filter_map(|((idx, _), layout)| {
-                if idx >= start_offset && idx < (child_len - 1 - end_offset) {
-                    Some((idx, layout.position().x + layout.bounds().width))
-                } else {
-                    None
+        let in_bounds = layout.bounds().contains(cursor_position);
+
+        if self.state.resizing || in_bounds {
```

Comment:
This is the only relevant part of the diff, everything else is just from getting moved into the if statement.